constitution.tex

```diff
@@ -275,18 +275,23 @@ \subsection{Modifying the Constitution}
 	\item Receive priority for on-floor housing, or apply for On-Floor Status
 \end{enumerate}
 
-\asubsubsection{Expectations of Voting Members}
+\asubsubsection{Qualifications of Voting Members}
 \renewcommand{\theenumi}{\arabic{enumi}} % For this section, we want items to use letters
 
-The following requirements must be met by the beginning of the Intro Eval for an Active Member to be allowed to vote.
-Any of these requirements may be waived by the Evals Director or an E-Board Vote.
+For an Active Member to be allowed to vote in an Intro Eval, they must meet the following requirements by the beginning of the Intro Eval.
+Any of these requirements may be waived by the Evals Director or an E-Board Vote. An Active Member who meets these requirements may opt out of Voting Membership by submitting a request to the Evals Director or E-Board.
 \begin{enumerate}
 	\item Attend all House Meetings during the Intro Process
 	\item Attend at least one directorship meeting for each week of the Intro Process % The number of directorship meetings intended here is equal to the number of the weeks in the process, not just one each week
 	\item Attend at least one CSH social event during the Intro Process
 	\item Attend at least two Technical Seminars during the Intro Process
 \end{enumerate}
 
+\asubsubsection{Expectations of Voting Members}
+\begin{enumerate}
+	\item Attend and vote at the Intro Eval for which they have qualified.
```

Comment:
formatting rec - merge into above list.

Reply:
These are two distinct lists with distinct meanings.

Reply:
I think they can be merged easily, and then we don't the very strange case where we have an enumerated list with one item.

For example, instead of 290-292, what if we put "Attend and vote at the Intro Eval, if eligible." at line 288?

Reply:
The first list is Qualifications; the second is expectations. Merging them doesn't make sense to me. I also don't see an issue with having a one-element list.